scripts/pylib/twister/twisterlib/testplan.py

```diff
@@ -562,11 +562,22 @@ def load_from_file(self, file, filter_platform=[]):
             instance_list = []
             for ts in jtp.get("testsuites", []):
                 logger.debug(f"loading {ts['name']}...")
-                testsuite = ts["name"]
+                testsuite = os.path.normpath(ts["name"])
 
                 platform = self.get_platform(ts["platform"])
                 if filter_platform and platform.name not in filter_platform:
                     continue
+
+                if testsuite not in self.testsuites:
+                # this could be a path issue in testplan and run envrionment
+                # try convert it
+                # so far we know window plan to linux envirnmont has such problem
+                    if "\\" in testsuite:
+                        testsuite = testsuite.replace("\\", "/")
+                    else:
```

Comment:
better to check once again `if testsuite not in self.testsuites:` instead of `else`. If there is no testsuite then we only replace backslashes, without warning (but with an exception in next lines)

Reply:
I think it is not necessary, as when we goes to this branch the `testsuite = os.path.normpath(ts["name"])` has done the job, so basically, this branch will only happen when normpath does not work, which is windows plan to Linux environment.